src/utils/util.js

```diff
@@ -40,7 +48,6 @@ module.exports = class Util {
 
       if (rate.time_in_transit[lowercaseDeliveryAccuracy] > parseInt(deliveryDays, 10)) {
         // eslint-disable-next-line no-continue
-        continue;
```

Comment:
Issue: Let's put this continue back, it tells the reader we want to move to the next iteration.

Reply:
Got removed in an IDE code cleanup